### DISCUSSION TITLE
Is it possible not to close tabby?

### DISCUSSION THREAD

**Discussion by RND332** [Eugeny/tabby#7595] [Q&A] [answered]
### Since it takes a long time to load, I would like tabby to just collapse into the tray

**Comment by Eugeny** [comment, marked as answer]
You can do this by using the "toggle-window" hotkey

**Comment by lawweiliang** [comment]
@RND332 
Please close the discussion if the question was answered. 
It would be helpful to make the discussion section clean. 

Click `mark as answered` and `close the discussion`. 😄 

![image](https://user-images.githubusercontent.com/34936929/228098333-222ac3d4-4c39-4c26-a2a8-52906647e00e.png)
